src/config.rs

```diff
@@ -189,6 +207,27 @@ mod test {
                     .unwrap();
             assert_eq!(gitlab, new_gitlab("user/repo", version("0.1.0")));
         }
+
+        #[test]
+        fn artifactory_from_artifactory_field() {
+            let artifactory: ToolSpec = toml::from_str(
+                &[
+                    r#"artifactory = "https://artifactory-edge1.rbx.com""#,
+                    r#"path = "generic-rbx-local-tools/rotriever/""#,
+                    r#"version = "0.5.4""#,
+                ]
+                .join("\n"),
+            )
+            .unwrap();
+            assert_eq!(
+                artifactory,
+                new_artifactory(
+                    "https://artifactory-edge1.rbx.com",
+                    "generic-rbx-local-tools/rotriever/",
+                    version("0.5.4"),
+                )
+            );
+        }
```

Comment:
Do we want to support leading and trailing backslashes in the artifactory and path sections such as 
```
[tools.rotrieve]
artifactory = "https://artifactory-edge1.rbx.com/"
path = "/generic-rbx-local-tools/rotriever/"
version = "0.5.4"
```

Reply:
I'm not really sure about this, it'd be nice to be flexible. I suspect when you join them together it doesn't matter too much if there's an extra slash, but it might also not matter at all if we use a Url class that has its own notions of "path" and "domain" and probably handles the rest for us?